### ISSUE TITLE
[Improvement] Change namespaces

### ISSUE THREAD

**Issue by Hri7566** [Nebual/sandbox-plus#38]
```c#
namespace SandboxPlus;
```

**Comment by Nebual** [comment]
Yes, this might also happen to allow us to be compatible with addons built against/referencing Facepunch/Sandbox (though there might also be plenty of other issues with that preventing that from ever really being a possibility)
